draft-yasskin-http-origin-signed-responses.md

```diff
@@ -949,9 +953,10 @@ to respond to. A PUSH_PROMISE (Section 8.2 of {{RFC7540}}) does not have this
 problem, and it would be possible to introduce a response header to convey the
 expected request headers.
 
-Since proxies don't modify unknown content types, we could wrap the original
-exchange into an `application/http2` format. This could be as simple as a series
-of HTTP/2 frames, or could
+Since proxies are unlikely to modify unknown content types, we could wrap the
+original exchange into an `application/http2` format and include the
+`Cache-Control: no-transform` header when sending it. This format could be as
```

Comment:
@yoavweiss I mentioned no-transform up here.

Reply:
oh, OK. missed it :/